src/Umbraco.Forms.Integrations.Automation.Zapier/Models/Dtos/FormConfigDto.cs

```diff
@@ -2,12 +2,15 @@
 
 namespace Umbraco.Forms.Integrations.Automation.Zapier.Models.Dtos
 {
-    public class SubscriptionDto
+    public class FormConfigDto
```

Comment:
Do we need this type now?  Was thinking we could just be using the type defined in the CMS integration package.

Reply:
Removed the dto and use only an array of strings, as the hook URL is the only value required on form submission.